fpm/src/fpm_command_line.f90

```diff
@@ -43,14 +46,17 @@ module fpm_command_line
 end type
 
 type, extends(fpm_run_settings)  :: fpm_test_settings
+integer :: gfortran_bug=0
```

Comment:
What is this `'gfortran_bug'` component used for?

Reply:
On an older machine with an old gfortran compiler on it would complain about defining a type with no elements.  Adding that line allowed for a build to work. I have since upgraded that machine and nothing I have (the oldest gfortran I now have is 8.3) produces an error so I will remove that.